2023-08-23 22:07:40,218:INFO-Image inserted , in handle_data.py line 24
2023-08-23 22:07:40,220:INFO-Sending the image for detection , in handle_data.py line 27
2023-08-23 22:07:42,153:INFO-Face is detected , in handle_data.py line 42
2023-08-23 22:07:42,154:INFO-Exception occured in handle_data.py line 69
2023-08-23 22:07:42,414:INFO-Image inserted , in handle_data.py line 24
2023-08-23 22:07:42,414:INFO-Sending the image for detection , in handle_data.py line 27
2023-08-23 22:07:42,916:INFO-Face is detected , in handle_data.py line 42
2023-08-23 22:07:42,916:INFO-Exception occured in handle_data.py line 69
2023-08-23 22:13:14,078:INFO-Trying to sigin from sigin.py line 15
2023-08-23 22:13:14,079:INFO-Creating cursor from sigin.py line 15
2023-08-23 22:19:55,545:INFO-Trying to signup from signup.py line 17
2023-08-23 22:19:55,546:INFO-Invalid credentials entered from signup.py line 35
2023-08-23 22:20:32,271:INFO-Trying to signup from signup.py line 17
2023-08-23 22:20:32,275:INFO-Invalid credentials entered from signup.py line 35
2023-08-23 22:23:21,853:INFO-Trying to signup from signup.py line 17
2023-08-23 22:23:21,855:INFO-Invalid credentials entered from signup.py line 35
2023-08-23 22:23:50,937:INFO-Trying to signup from signup.py line 17
2023-08-23 22:23:50,938:INFO-Invalid credentials entered from signup.py line 35
2023-08-23 22:24:17,778:INFO-Trying to signup from signup.py line 17
2023-08-23 22:24:17,779:INFO-Invalid credentials entered from signup.py line 35
2023-08-23 22:24:40,873:INFO-Trying to signup from signup.py line 17
2023-08-23 22:24:40,874:INFO-Invalid credentials entered from signup.py line 35
2023-08-23 22:25:01,167:INFO-Trying to signup from signup.py line 17
2023-08-23 22:25:01,168:INFO-Invalid credentials entered from signup.py line 35
2023-08-23 22:25:11,320:INFO-Trying to signup from signup.py line 17
2023-08-23 22:25:11,320:INFO-Invalid credentials entered from signup.py line 35
2023-08-23 22:25:34,215:INFO-Trying to signup from signup.py line 17
2023-08-23 22:25:34,216:INFO-Invalid credentials entered from signup.py line 35
2023-08-23 22:25:54,496:INFO-Trying to signup from signup.py line 17
2023-08-23 22:25:54,496:INFO-Invalid credentials entered from signup.py line 35
2023-08-23 22:30:50,523:INFO-Trying to signup from signup.py line 17
2023-08-23 22:30:50,525:INFO-Valid credentails entered from signup.py line 19
2023-08-23 22:30:50,525:INFO-Cursor created from signup.py line 22
2023-08-23 22:30:50,559:INFO-Query executed from signup.py line 26
2023-08-23 22:31:18,677:INFO-Trying to signup from signup.py line 17
2023-08-23 22:31:18,687:INFO-Valid credentails entered from signup.py line 19
2023-08-23 23:57:00,333:INFO-Trying to signup from signup.py line 17
2023-08-23 23:57:00,334:INFO-Invalid credentials entered from signup.py line 35
2023-08-23 23:57:17,846:INFO-Trying to signup from signup.py line 17
2023-08-23 23:57:17,846:INFO-Valid credentails entered from signup.py line 19
2023-08-23 23:57:17,846:INFO-Cursor created from signup.py line 22
2023-08-23 23:57:17,865:INFO-Query executed from signup.py line 26
